### DISCUSSION TITLE
Should languages be limited to a certain % translated before being available?

### DISCUSSION THREAD

**Discussion by Tzahi12345** [Tzahi12345/YoutubeDL-Material#865] [Ideas]
There are some languages that haven't been updated for a while, so maybe only 20-30% of strings are translated. Is it better than nothing to have these available or misleading?

Maybe for those less than 80% translated we show a warning, and anything below 20% we hide.

**Comment by GlassedSilver** [comment]
Probably a good idea to show the warning with a graphical icon, since it'd be ironical for unmaintained languages to not get the warning. :D 

I say, offer them, but warn that the translation is very lacking. I think anything below 60% is critical.
20% is a started effort. Still makes sense to offer anything we have really, because if you only interact with the app sporadically and mostly leave it running unattended you may get from it what you need with whatever is translated and be grateful for it. If the rest is filled up with English and you can get by translating some bits I'm sure it's more helpful than having to translate even the very common ones that WOULD be translated now.
